src/Source/WebformSubmissionCount.php

```diff
@@ -24,28 +24,72 @@ public static function forField($entity, $field, $instance) {
   public function __construct($entity) {
     $this->entity = $entity;
   }
+
   /**
    * Get the value for the given item.
    *
    * @return int
-   *   The number of webform submissions in $this-entity
-   *   and all it's translations.
+   *   The number of webform submissions in $this-entity,
+   *   and on additional nodes provided via the field widget,
+   *   and all their translations.
    */
   public function getValue($item) {
     $entity = $this->entity;
-    $q = db_select('node', 'n');
+    $nids = array($entity->nid);
+    if (isset($item['options']['source']['add_nids'])
+        && $add_nids = $item['options']['source']['add_nids']) {
+      array_push($nids, ...$add_nids);
```

Comment:
[The ... syntax](http://php.net/manual/en/functions.arguments.php#functions.variable-arg-list) was introduced in PHP 5.6. Currently we only require PHP 5.4 (new-style array syntax). We could bump the requirement to PHP 5.6 but should do so explicitly.